### ISSUE TITLE
Run the kNN for multiple number of neighbors

### ISSUE THREAD

**Issue by karlvmsousa** [karlvandesman/IN1102-Machine-Learning#8]
In the main.py, would be a good choice run the kNN for different values of n_neighbors, and select the best choice.

**Comment by karlvmsousa** [comment]
For the training set: 
`n_neighbors = 14`

For the test set:
`n_neighbors = 9`

![tuning_knn](https://user-images.githubusercontent.com/31048109/69744794-a221bd80-111f-11ea-9640-55da1fe32a45.png)
